src/main/java/org/gradle/profiler/GradleBuildInvocationResult.java

```diff
@@ -43,8 +45,8 @@ public Duration getTimeToTaskExecution() {
         return timeToTaskExecution;
     }
 
-    public Map<String, Duration> getCumulativeBuildOperationTimes() {
-        return cumulativeBuildOperationTimes;
+    public Map<String, BuildOperationDuration> getCumulativeBuildOperationDurations() {
+        return cumulativeBuildOperationDurations;
```

Comment:
We use "total" in other places to refer to the same concept we use "cumulative" to describe here. Would be nice to use the same.

Reply:
Done